src/Avalonia.Themes.Fluent/Controls/ProgressBar.xaml

```diff
@@ -29,7 +30,11 @@
                 <Border x:Name="IndeterminateProgressBarIndicator" CornerRadius="{TemplateBinding CornerRadius}" Margin="{TemplateBinding Padding}" Background="{TemplateBinding Foreground}" />
                 <Border x:Name="IndeterminateProgressBarIndicator2" CornerRadius="{TemplateBinding CornerRadius}" Margin="{TemplateBinding Padding}" Background="{TemplateBinding Foreground}" />
               </Panel>
-            </Panel>          
+            </Panel>
+            <LayoutTransformControl HorizontalAlignment="Center" VerticalAlignment="Center" IsVisible="{Binding ShowProgressText, RelativeSource={RelativeSource TemplatedParent}}" x:Name="PART_LayoutTransformControl">
+              <TextBlock Foreground="{DynamicResource ThemeForegroundBrush}" Text="{Binding Value, RelativeSource={RelativeSource TemplatedParent}, StringFormat={}{0:0}%}" />
```

Comment:
ThemeForegroundBrush resource doesn't exist in Fluent theme.

Reply:
> > A new property should be added to disable this without re-templating.
> 
> Why? ShowProgressText is already there and disabled by default.
> 
> @Abdesol default progress bar is somewhat 4px height, right? Do you need to adjust height as well if you set ShowProgressText to true or is it automatically done by current layout?

It is automatically done by the current layout.
See the below images. The height of the progress bar is not set so it is the default one but it is increasing on ShowProgressText = true.

ShowProgressText = false
![image](https://user-images.githubusercontent.com/69455299/147802419-8aa687c0-0b9a-493c-93ff-09f55faaab5c.png)


ShowProgressText = true
![image](https://user-images.githubusercontent.com/69455299/147802428-b3a0a773-b407-4a6d-86a8-e67226b0a7d7.png)

Reply:
> ThemeForegroundBrush resource doesn't exist in Fluent theme.

Ohok... Fixed it now by changing it to `Foreground="{TemplateBinding Foreground}"`